exports/request_resources.go

```diff
@@ -33,21 +35,46 @@ func KafkaRequestApplicationResources(kafkaChan chan *kafka.Message) RequestAppl
 			}
 
 			for _, source := range sources {
+				filters, err := ekafka.JsonToInterface(source.Filters)
+				if err != nil {
+					log.Errorw("failed unmarshalling filters", "error", err)
+					// FIXME:
+					// return err
+					return
+				}
+
+				format, ok := ekafka.ParseFormat(string(payload.Format))
+				if !ok {
+					log.Errorw("failed parsing format", "error", err)
+					// FIXME:
```

Comment:
Currently, these errors are only logged. In the future it may be a good idea to create a seperate channel for handling errors that occur here.